python/dgl/distributed/graph_partition_book.py

```diff
@@ -623,6 +623,9 @@ def partid(self):
         """
         return self._partid
 
+NODE_PART_POLICY = 'node'
```

Comment:
Why we need this?

Reply:
we need different partition policies for nodes and edges. i just create "constants" for the two policies.